code/extensions/UserDefinedFormFreshdeskExtension.php

```diff
@@ -12,4 +12,17 @@ public function updateCMSFields(FieldList $fields)
         $fields->addFieldToTab('Root.FormOptions', new CheckboxField('ExportToFreshdesk', 'Export as a Freshdesk ticket on submit'));
         $fields->addFieldToTab('Root.FormOptions', new TextareaField('FreshdeskDescription', 'Freshdesk ticket description:'));
     }
+
+    public function validate(ValidationResult $validationResult)
+    {
+        parent::validate($validationResult);
+
+        if (!$this->owner->ExportToFreshdesk) {
+            return $validationResult->valid();
```

Comment:
This extension hook does not require a return value. See `DataObject::validate()`.

Also, you don't need to call `valid`, because that's the default. The whole conditional can likely be removed.